demos/canvas-renderer-matplotlib/demo-1/demo.js

```diff
@@ -0,0 +1,37 @@
+async function main() {
+  let pyodide = await loadPyodide({
+    indexURL: "https://cdn.jsdelivr.net/pyodide/v0.19.1/full/",
+  });
```

Comment:
```suggestion
  let pyodide = await loadPyodide();
```
it's no longer necessary with 0.20.0a1

Reply:
done, thanks